packages/mystmd-py/pyproject.toml

```diff
@@ -34,6 +34,14 @@ myst = "mystmd_py.main:main"
 source = "nodejs"
 path = "_package.json"		
 
+# Ensure that we only bring in nodeenv for PyPI
+[tool.hatch.build.hooks.custom]
+path = "build_plugin.py"
+pypi_dependencies = [
+  "platformdirs~=4.2.2",
+  "nodeenv~=1.9.1"
+]
+
```

Comment:
@rowanc1 this is a build plugin for hatch that injects the PyPI dependencies at build time. The conda-forge build pulls in the sdist (which is mainly just a tar of the package contents), so it does not have this injected content. When the conda-forge build happens, we'll need to update the environment to set MYST_IS_CONDA_PACKAGE, which will disable the injection.